makeBeamCurrentFileII.py

```diff
@@ -56,12 +56,19 @@ def checkFBCTcalib(table, CalibrateFBCTtoDCCT):
 
         print "Applying FBCT to DCCT calibration"
         for idx, entry in enumerate(table):
-            old1 = entry[7]
-#            entry[7] = entry[5]/entry[3]*old1
-            entry[7] = corrB1*old1
-            old2 =  entry[8]
-#            entry[8] = entry[6]/entry[4]*old2
-            entry[8] = corrB2*old2
+            K1=entry[5]/entry[3]
```

Comment:
I think you should still take the parameters from the fit and propagate them.  What you have now will take the average and I don't think it solves your problem.  

To do this generically, I recommend 
1) passing a parameter for the fit model (if pol0: fitM = ROOT.TF1("pol0"), elif pol1:  fitM= etc) 
2) fit the DCCT/FBCT data  (ratioHist.Fit(fitM))
3) apply (see next comment)